### PR TITLE
#AnimalGenomics updates

### PR DESCRIPTION
**Pull request by bjlang**

### CODE REVIEW

File: sites/main-site/src/content/events/2024/SIG_animalgenomics_Sept.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+endTime: '17:00+02:00'
+locations:
+  - name: Online
+    links:
+      - https://rediris.zoom.us/j/96065549412?pwd=L2txanpQMEFTOWVLNmtIZyt6M3NnUT09
+---
+
+September 2024 talk of the monthly meeting of the _#animal-genomics_ special interest group.
+In this talk, **Guillaume Devailly** (INRAE, France) will present **Automatic visualisation of FAANG data**:
+_<ul>The FAANG data portal offers a great curation of transcriptomic and epigenomic data from livestock species. We would like to improve FAANG data re-use by offering automatic visualisations of available datasets: sample - sample correlation heatmaps, genomic stacks at interesting features (peak center, transcription start site, CpG island, ...). We therefore need to automatically process FAANG dataset using a Nextflow pipeline and could provide uniformly processed files and QC reports as useful artifacts. While an initial protoype is available, help is needed to sustain this development.</ul>_
```
Comment (mashehu) [inline sites/main-site/src/content/events/2024/SIG_animalgenomics_Sept.md:18]: Why the unordered list tag? I would prefer to keep it purely markdown.

Comment (bjlang) [reply sites/main-site/src/content/events/2024/SIG_animalgenomics_Sept.md:18]: I wanted some kind of indentation as visual separation of the abstract, but would prefer it without the vertical line that a block quote would give. Of course it's not really important, if pure markdown should be used.

Comment (mashehu) [reply sites/main-site/src/content/events/2024/SIG_animalgenomics_Sept.md:18]: Okay, wrapping it in `<div class="mx-2">` feels less hacky tbh.

File: sites/main-site/src/content/events/2024/SIG_animalgenomics_Sept.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+endTime: '17:00+02:00'
+locations:
+  - name: Online
+    links:
+      - https://rediris.zoom.us/j/96065549412?pwd=L2txanpQMEFTOWVLNmtIZyt6M3NnUT09
+---
+
+September 2024 talk of the monthly meeting of the _#animal-genomics_ special interest group.
+In this talk, **Guillaume Devailly** (INRAE, France) will present **Automatic visualisation of FAANG data**:
+_<div class="mx-2">The FAANG data portal offers a great curation of transcriptomic and epigenomic data from livestock species. We would like to improve FAANG data re-use by offering automatic visualisations of available datasets: sample - sample correlation heatmaps, genomic stacks at interesting features (peak center, transcription start site, CpG island, ...). We therefore need to automatically process FAANG dataset using a Nextflow pipeline and could provide uniformly processed files and QC reports as useful artifacts. While an initial protoype is available, help is needed to sustain this development.</div>_
```
Comment (mashehu) [inline sites/main-site/src/content/events/2024/SIG_animalgenomics_Sept.md:18]:
```suggestion
<div class="mx-2">The FAANG data portal offers a great curation of transcriptomic and epigenomic data from livestock species. We would like to improve FAANG data re-use by offering automatic visualisations of available datasets: sample - sample correlation heatmaps, genomic stacks at interesting features (peak center, transcription start site, CpG island, ...). We therefore need to automatically process FAANG dataset using a Nextflow pipeline and could provide uniformly processed files and QC reports as useful artifacts. While an initial protoype is available, help is needed to sustain this development.</div>
```
I find long italic text difficult to read. but that's probably just a question of taste, so ignore if you like it.

**Review by mashehu — APPROVED** [review]

**Pull request merged**